content/en/tabcontents.yaml

```diff
@@ -89,7 +89,7 @@ params:
       alttext: A computer chip.
       img: /images/content_images/sc_dom_img/quantum_computing.svg
       links:
-        - url: http://qutip.org
+        - url: https://qutip.org
```

Comment:
@rgommers @steppi It looks like this isn't in `content/pt/tabcontents.yaml` or `content/ja/tabcontents.yaml`. Is there a plan or mechanism to keep the translations up-to-date?

Reply:
@steppi could you please have a look at this?